ci/Jenkinsfile.linux

```diff
@@ -57,7 +57,7 @@ pipeline {
           credentialsId: utils.getInfuraTokenCred(),
           variable: 'INFURA_TOKEN'
         )]) {
-          sh 'make nim_status_client'
+          sh 'make INFURA_TOKEN=$INFURA_TOKEN nim_status_client'
```

Comment:
We can drop the `INFURA_TOKEN=$INFURA_TOKEN` part.

Reply:
Are you sure? I'll give the equivalent a shot locally but I've encountered situations where just because an environment variable with the same name as a `make` variable is defined, `make` didn't honor it unless you explicitly pass the variable to `make`, i.e. on the RHS of `make. But... I may have misunderstood what was going on.

Reply:
Alright, my mistake. There's probably a time I accidentally did something like:
```
$ FOO=123
$ make target
```
instead of
```
$ FOO=123 \
make target
```
or
```
$ export FOO=123
$ make target
```
And therefore mistook `?=` to be require variable overrides on the RHS of `make` as do `:=` and `=` assignments in Makefiles.